components/CampaignListItem/index.tsx

```diff
@@ -0,0 +1,121 @@
+import ArchiveOutlinedIcon from '@mui/icons-material/ArchiveOutlined'
+import { Box } from '@mui/material'
+
+const styledmainbox = {
+  display: 'flex',
+  flexDirection: 'column',
+  width: '1070px',
+  height: '100%',
+  padding: '10px',
+  backgroundColor: '#EAF9FF',
+}
+
+const styledmainbody = {
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'space-between',
+  padding: '0px 20px 0px 20px',
+}
+
+const styledcontentbox = {
+  display: 'flex',
+  height: '300px',
+  backgroundColor: '#D9D9D9',
+}
+
+const styledbottombox = {
+  display: 'flex',
+  flex: 'auto',
+  flexDirection: 'row',
+  justifyContent: 'space-between',
+  alignItems: 'center',
+  width: '100%',
+  height: '67px',
+  backgroundColor: '#FFFFFF',
+  padding: '0px 20px 0px 20px',
+}
+
+const styledbottomitems = {
+  display: 'flex',
+}
+
+const styledinput = {
+  fontSize: '15',
+  fontWeight: '700',
+}
+
+type Props = {
+  CampaignName: String
+  CampaignDetails: String
+  Donors: Number
+  Amount: Number
+  Country: String
+  Published: Boolean
```

Comment:
use camelCase for all variable names and types should be in lowercase ie string instead of String, number instead of Number etc. recommended to use interface instead of type for object kind of types.
```
type Props = {
  campaignName: string
  campaignDetails: string
  donors: number
  amount: number
  country: string
  campaignStatus: CampaignStatus
  viewAll: boolean
}
```